processor/transaction/package_tests/TestProcessTransactionNullValues.approved.json

```diff
@@ -3,6 +3,7 @@
         {
             "@timestamp": "2017-05-30T18:53:27.154Z",
             "context": {
+                "process": {},
```

Comment:
same comment as above - empty `process` does not reflect spec, where `pid` is required.